src/components/tab/bl-tab.css

```diff
@@ -0,0 +1,84 @@
+/*
+ Avoid empty file error in linter
+*/
+
+:host {
+    position: relative;
+    display: flex;
+    align-items: center;
+    background-color: var(--bl-color-primary-background);
+
+    --bl-tab-title-padding-vertical: var(--bl-size-m);
+    --bl-tab-title-padding-horizontal: var(--bl-size-xl);
+    --bl-tab-title-color: var(--bl-color-secondary);
```

Comment:
CSS files should also have 2 space per tab. Normally we have `.editorconfig` file but maybe you can check an extension for your IDE that will respect the configurations in that file.